src/types/global.d.ts

```diff
@@ -0,0 +1,12 @@
+export {};
+
+declare global {
+    interface ShadowRoot {
+        adoptedStyleSheets: CSSStyleSheet[];
```

Comment:
nit: we should use the same type as `document.styleSheets` here. 

I think we could also move this to the existing `globals.d.ts` file?